<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserViewer</class>
 <widget class="QWidget" name="UserViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="group_budgets">
     <property name="title">
      <string>Budgets</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="budgets_list">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="budgets_buttons">
        <item>
         <widget class="QPushButton" name="add_budget">
          <property name="text">
           <string> Add</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/playlist-plus.svg</normaloff>:/resources/material-design/svg/playlist-plus.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_budget">
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/pencil.svg</normaloff>:/resources/material-design/svg/pencil.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_budget">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/playlist-remove.svg</normaloff>:/resources/material-design/svg/playlist-remove.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_accounts">
     <property name="title">
      <string>Accounts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QListWidget" name="accounts_list">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="accounts_buttons">
        <item>
         <widget class="QPushButton" name="add_account">
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/playlist-plus.svg</normaloff>:/resources/material-design/svg/playlist-plus.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_account">
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/pencil.svg</normaloff>:/resources/material-design/svg/pencil.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_account">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../../picsou.qrc">
            <normaloff>:/resources/material-design/svg/playlist-remove.svg</normaloff>:/resources/material-design/svg/playlist-remove.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../picsou.qrc"/>
 </resources>
 <connections/>
</ui>
